src/pages/stacks-blockchain/technical-specs.md

```diff
@@ -0,0 +1,52 @@
+- Delegation: An STX address can designate another address to participate in Stacking on its behalf.
+- Pooling: STX holders that individually do not meet the Stacking threshold can pool together their holdings to participate in Stacking. To do this, STX holders must set the (optional) reward address to the "delegate address". For more details, see [this reference](https://docs.blockstack.org/references/stacking-contract#delegate-stx).
+
+## Accounts and Addresses
+
+- Transactions in the Stacks blockchain originate from, are paid for by, and execute under the authority of accounts
+- An account is fully specified by its address + nonce + assets
+- Address contains 2 or 3 fields: 1 byte version, 20 byte public key hash (RIPEMD160(SHA256(input))), optional name (variable length, max 128 bytes)
+- Two types of accounts: standard accounts are owned by one or more private keys; contract accounts are materialized when a smart-contract is instantiated (specified by the optional name field above)
+- Nonce counts number of times an account has authorized a transaction. Starts at 0, valid authorization must include the _next_ nonce value.
+- Assets are a map of all asset types -- STX, any on-chain assets specified by a Clarity contract (e.g. NFTs) -- to quantities owned by that account.
+- Accounts need not be explicit "created" or registered; all accounts implicitly exist and are instantiated on first-use.
+
+## Transactions
+
+- [Transaction types](https://docs.blockstack.org/stacks-blockchain/transactions#types): coinbase, token-transfer, contract-deploy, contract-call, poison-microblock.
+- Only standard accounts (not contracts) can pay transaction fees.
+- Transaction execution is governed by 3 accounts (may or may not be distinct)
+  1. _originating account_ is the account that creates, _authorizes_ and sends the transaction
+  2. _paying account_ is the account that is billed by the leader for the cost of validating and executing the transaction
+  3. _sending account_ is the account that identifies who is currently executing the transaction: this can change as a transaction executes via the `as-contract` Clarity function
+- Two types of authorizations: standard authorization is where originating account is the same as paying account. _Sponsored_ authorization is where originating account and paying account are distinct. For instance, developers or service providers could pay for users to call their smart-contracts.
+- For sponsored authorization, first a user signs with the originating account and then a sponsor signs with the paying account.
+- Transaction encoding is described [here](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-encoding).
+- Transaction signing and verification are described [here](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md#transaction-signing-and-verifying).
```

Comment:
https://github.com/blockstack/docs/blob/master/src/pages/stacks-blockchain/transactions.md#signature-and-verification